server/@types/manage-recalls-api/models/RecallResponse.ts

```diff
@@ -55,7 +55,7 @@ export type RecallResponse = {
     sentenceExpiryDate?: string;
     sentenceLength?: SentenceLengthRes;
     sentencingCourt?: string;
-    status?: RecallResponse.status;
+    status: RecallResponse.status;
```

Comment:
maybe worth updating the get-recall.json response in fake-manage-recalls-api

Reply:
yep, that was causing the failure :(